tests/test_evaluations/test_evaluation_functions.py

```diff
@@ -142,3 +142,57 @@ def test_list_evaluation_measures(self):
         measures = openml.evaluations.list_evaluation_measures()
         self.assertEqual(isinstance(measures, list), True)
         self.assertEqual(all([isinstance(s, str) for s in measures]), True)
+
+    def test_list_evaluations_setups_filter_flow(self):
+        openml.config.server = self.production_server
+        flow_id = 405
+        size = 100
+        evals_setups = openml.evaluations.list_evaluations_setups("predictive_accuracy",
```

Comment:
I know we're generally not very good at code re-use, particularly in unit tests, but I'd rather change that :) Please extract the common code between `test_list_evaluations_setups_filter_flow` and `test_list_evaluations_setups_filter_task` into one function that is called from both tests (e.g. specifying either `flow=[flow_id]` or `task=[task_id]` through `**kwargs`).